packages/core/src/components/info-button/info-button.scss

```
@@ -12,6 +12,8 @@
 
   --di-info: #{dso-icon("info")};
   --di-info-active: #{dso-icon("info-active")};
+  --di-info-secondary: #{dso-icon("info", "secondary")};
+  --di-info-active-secondary: #{dso-icon("info-active", "secondary")};
```

Je hanteert twee notaties:
* `component-status-variant`
* `component-variant-status`.

https://github.com/dso-toolkit/dso-toolkit/pull/1350/files#diff-cf732901b068593ab4c9370b37b4873d83548766ccf71096ec17833e02c81d18R19

Ik heb voorkeur voor de laatste, dus deze file moet aangepast worden:

```suggestion
  --di-info-secondary-active: #{dso-icon("info-active", "secondary")};
```

---

Ik weet niet of dat kan? Hij luistert naar `--di-info-active-secondary` waarschijnlijk doordat de notatie in `info-active.scss` zo staat:

```
info-active:secondary {
  color: $grijs-60;
}
```

---

Ah, je hebt gelijk: De variable is niet voor het info component maar voor het **d**so **i**con component. Goed gevonden. We laten het zo